addons/hr_holidays/static/tests/thread_patch.test.js

```diff
@@ -9,7 +9,14 @@ defineHrHolidaysModels();
 test("out of office message on direct chat with out of office partner", async () => {
     const pyEnv = await startServer();
     const partnerId = pyEnv["res.partner"].create({ name: "Demo", im_status: "online" });
-    pyEnv["res.users"].create({ partner_id: partnerId, leave_date_to: "2023-01-01" });
+    const userId = pyEnv["res.users"].create({ partner_id: partnerId });
+    const employee = pyEnv["hr.employee"].create({
+        user_id: userId,
+        leave_date_to: "2023-01-01",
```

Comment:
This might be asking too much for now, but ideally this should be computed by the mock server based on leave records, rather than being hard-coded here.